lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1374,6 +1381,28 @@ MemOp::PortKind MemOp::getPortKind(size_t resultNo) {
       getResult(resultNo).getType().cast<FIRRTLType>());
 }
 
+/// Return the number of bits in the mask for the memory.
+size_t MemOp::getMaskBits() {
+  assert(getNumResults() != 0 && "Mems with no read/write ports are illegal");
```

Comment:
should be handled in verification.

Reply:
Removed this and added a relevant check to verification.